packages/main/src/TableHeaderRow.ts

```diff
@@ -97,6 +99,10 @@ class TableHeaderRow extends TableRowBase {
 	get _i18nRowPopin() {
 		return TableRowBase.i18nBundle.getText(TABLE_ROW_POPIN);
 	}
+
+	get _cells() {
```

Comment:
who will remember to update this method if we add a new cell?
I do expect something more general e.g. see e.g. TableNavigation#_getNavigationItemsOfRow

Reply:
Obsolete method now. Moved it into TableRowBase